eldritch/ancient_ones.py

```diff
@@ -230,7 +232,146 @@ def escalate(self, state):
+        continue
+      check = events.Check(char, "sneak", self.sneak_modifier, name=self.name)
+      checks.append(
+          events.PassFail(
+              char, check, events.Nothing(),
+              events.BinarySpend(
+                  char, "monsters", 1, "Lose one monster trophy or be devoured",
+                  "Spend", "Be Devoured",
+                  events.Nothing(), events.Devoured(char)
+              )
+          )
+      )
+    return AncientOneAttack(checks)
+
+  def escalate(self, state):
+    self.sneak_modifier -= 1
+
+
+class SerpentGod(AncientOne):
+  def __init__(self):
+    super().__init__("Serpent God", 10, {}, -3)
+    self.speed_modifier = 1
+
+  def get_modifier(self, thing, attribute, state):
+    if isinstance(thing, monsters.Cultist):
+      if attribute == "combat_difficulty":
+        return -1
+      if attribute == "combat_damage":
+        return 3
+    return 0
+
+  def get_trigger(self, event, state):
+    if isinstance(event, events.LostInTimeAndSpace):
+      return events.AddToken(self, "doom")
+    if (
+        isinstance(event, events.PassCombatRound)
+        and isinstance(event.combat_round.monster, monsters.Cultist)
+    ):
+      return events.AddDoom(character=event.character)
+    if isinstance(event, events.Awaken):
+      per_character = []
+      for char in state.characters:
+        assert isinstance(char, characters.Character)
+        if char.gone:
+          continue
+        if char.bless_curse == -1:
+          per_character.append(events.Devoured(char))
+        else:
+          per_character.append(events.Curse(char))
+      return events.Sequence(per_character)
+
+    return None
+
+  def attack(self, state):
+    checks = []
+    for char in state.characters:
+      if char.gone:
+        continue
+      check = events.Check(char, "speed", self.speed_modifier, name=self.name)
+      checks.append(
+          events.PassFail(
+              char, check,
+              events.Nothing(), events.Loss(char, {"sanity": 1, "stamina": 1})
+          )
+      )
+    return AncientOneAttack(checks)
+
+  def escalate(self, state):
+    self.speed_modifier -= 1
+
+
+class SpaceBubbles(AncientOne):
+  def __init__(self):
+    super().__init__("Space Bubbles", 12, {"magical_immunity"}, -5)
+    self.will_modifier = 1
+
+  def get_trigger(self, event, state):
+    if isinstance(event, events.Awaken):
+      to_devour = []
+      for char in state.characters:
+        assert isinstance(char, characters.Character)
+        if char.n_gate_trophies == 0:
+          to_devour.append(events.Devoured(char))
+      return events.Sequence(to_devour)
+    if isinstance(event, events.LostInTimeAndSpace):
+      return events.Devoured(event.character)
+    return None
+
+  def attack(self, state):
+    checks = []
+    for char in state.characters:
+      if char.gone:
+        continue
+      check = events.Check(char, "speed", self.will_modifier, name=self.name)
+      checks.append(
```

Comment:
You seem to have a list of checks, to which you append a sequence with only one element in it, and then do nothing with the checks?